src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -202,16 +202,54 @@ export const generateNavItems = (): NavItem[] => {
       link: `/operator`,
       activeKeywords: ['operator'],
     },
-    {
-      id: 'support',
-      testID: 'nav-item-support',
-      icon: IconFont.QuestionMark_New,
-      label: 'Help & Support',
-      shortLabel: 'Support',
-      link: '',
-      activeKeywords: [''],
-      enabled: () => isFlagEnabled('helpBar'),
-    },
+    // {
```

Comment:
It looks like you're rendering this below and this object in the array is not being used. Should we remove it from the pr?